frontend/src/pages/Workflows/WorkflowDetails.js

```diff
@@ -144,6 +145,12 @@ function WorkflowDetails({
     },
     [showWorkflowDetails]
   );
+  useEffect(
```

Comment:
The effect is called every time, even if it is not necessary. I believe this hook has to move into the component itself.

Reply:
changed implementation of `<WorkflowitemHistoryTab>` to class based and used `componentWillUnmount` to reset state